### PR TITLE
[MP-194] Improve Unity WebGL wallet control

### PR DESCRIPTION
**Pull request by sidorovkirill**

### CODE REVIEW

File: Assets/AnkrSDK/Scripts/Core/Infrastructure/IAnkrSDK.cs [hunk 1]
```diff
@@ -6,5 +9,6 @@ public interface IAnkrSDK
 		INetworkHelper NetworkHelper { get; }
 		IContract GetContract(string contractAddress, string contractABI);
 		IContractEventSubscriber CreateSubscriber(string wsUrl);
+		UniTask<Dictionary<string, bool>> GetWalletsStatus();
```
Comment (Vukiz) [inline Assets/AnkrSDK/Scripts/Core/Infrastructure/IAnkrSDK.cs:12]: Lets expose IWalletHandler instead of its one method. This will fit better here

File: Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
 		private readonly WalletConnect _walletConnect;
-
-		public MobileDisconnectHandler()
+ 
+		public MobileWalletHandler()
 		{
 			_walletConnect = ConnectProvider<WalletConnect>.GetWalletConnect();
 		}
 
+		public UniTask<Dictionary<string, bool>> GetWalletsStatus()
```
Comment (Vukiz) [inline Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs:18]: Create a named class for <string, bool> tuple

File: Assets/AnkrSDK/Scripts/WebGL/WebGLCommunicationProtocol.cs [hunk 3]
```diff
@@ -37,7 +37,7 @@ public string GenerateId()
 			
 			var completionTask = new UniTaskCompletionSource<WebGLMessageDTO>();
 			_completionSources.Add(id, completionTask);
-			
+
```
Comment (Vukiz) [inline Assets/AnkrSDK/Scripts/WebGL/WebGLCommunicationProtocol.cs:40]: unnecessary change

File: Assets/AnkrSDK/Scripts/WebGL/WebGLCommunicationProtocol.cs [hunk 4]
```diff
@@ -46,7 +46,7 @@ public async UniTask<WebGLMessageDTO> WaitForAnswer(string id)
 			var completionTask = _completionSources[id];
 			var answer = await completionTask.Task;
 			_completionSources.Remove(id);
-
+
```
Comment (Vukiz) [inline Assets/AnkrSDK/Scripts/WebGL/WebGLCommunicationProtocol.cs:49]: unnecessary change

File: Assets/AnkrSDK/Scripts/WebGL/WebGLConnect.cs [hunk 5]
```diff
@@ -33,6 +35,7 @@ private async void Start()
 			DontDestroyOnLoad(this);
```
Comment (Vukiz) [inline Assets/AnkrSDK/Scripts/WebGL/WebGLConnect.cs:35]: Move DontDestroyOnLoad(this); inside if statement also

File: Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs (hunk 2 above)
Comment (Vukiz) [reply Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs:18]: Put a dictionary inside that class. Something like
WalletsStatusData{
public Dictionary< WalletStatusData >

}
WalletStatusData{
public string
public bool
}

**Review by Vukiz — CHANGES_REQUESTED** [review]

File: Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs (hunk 2 above)
Comment (sidorovkirill) [reply Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs:18]: As I know Dictionaty need key and value types. In that case we have only way like that
`public class WalletStatusData : Dictionary<string, bool> {}`
or I need to change JS lib

**Review by Vukiz — APPROVED** [review]

**Comment by Vukiz** [comment]
Please add a task to improve this later
